src/presentational/button/index.tsx

```
@@ -0,0 +1,37 @@
+import * as React from "react";
+import styled from "styled-components";
+import { colors, paywallColors, fonts } from "../../theme/index";
+
+interface DynamicStyleProps {}
+
+const Button = styled.button<DynamicStyleProps>`
+  width: 100%;
+  max-width: 270px;
+  padding-top: 17px;
+  padding-bottom: 12px;
+  background-color: ${paywallColors.timesGreen};
+  font-family: ${fonts.gillSansMedium};
+  color: ${colors.mainWhite};
+  font-size: 15px;
+  line-height: 15px;
+  border: 0;
+  border-radius: 5px;
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  @media (max-width: 719px) {
+    max-width: 100%;
+  }
+`;
+
+type Props = {
+  children: string;
+};
+
+const DynamicPriceMessaging: React.FC<Props> = props => (
```

defaults plz

---

This component should fail if children property is undefined, so let's not define defaults